server/gc/src/main/java/org/apache/accumulo/gc/GCRun.java

```diff
@@ -462,4 +479,28 @@ public long getErrorsStat() {
   public long getCandidatesStat() {
     return candidates;
   }
+
+  @Override
+  public boolean isRootTable() {
+    return level == DataLevel.ROOT;
+  }
+
+  @Override
+  public boolean isMetadataTable() {
+    return level == DataLevel.METADATA;
+  }
```

Comment:
It would be nice to remove these if possible, seems like they are only used in test.  Could just make test override/implement the method to get table ids.  If it is needed for test I would suggest adding a getDataLevel() method.

```suggestion
```